process/factory/metachain/preProcessorsContainerFactory.go

```diff
@@ -127,21 +127,6 @@ func NewPreProcessorsContainerFactory(
 	if check.IfNil(txExecutionOrderHandler) {
 		return nil, process.ErrNilTxExecutionOrderHandler
 	}
```

Comment:
We don't test (verify) the passed config here - we verify in the constructor of the component where the configuration entries are read / used.